docs/content/index.mdx

```diff
@@ -44,6 +44,16 @@ If using SAM, you can include this SAR App as part of your shared Layers stack,
         SemanticVersion: 1.3.1 # change to latest semantic version available in SAR
 ```
 
+This will add a nested app stack with an output parameter `LayerVersionArn`. You can not reference layer arn within the template on the first try, this is currently not supported.
```

Comment:
Why is it not supported?

How did you try to reference it?

Reply:
I tried to reference it via !Ref and !GetAtt via the logical resource name within the template:

```
   Layers:
     - !GetAtt AwsLambdaPowertoolsPythonLayer.Arn
```

going via outputs as you have showed in your example is working:

```
   Layers:
     - !GetAtt AwsLambdaPowertoolsPythonLayer.Outputs.LayerVersionArn
```

I have updated the docs and added this to the `example` project we have under `template.yml`